spec/features/comments_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature 'Leaving comments' do
+  background do
+    visit '/'
+    sign_up(username: 'allimac',email: 'camilla@email.com', password: 'pass1234')
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)